images/win/scripts/Tests/BizTalk.Tests.ps1

```diff
@@ -0,0 +1,37 @@
+################################################################################
+##  File:  BizTalk.Tests.ps1
+##  Desc:  Test BizTalk project build component installed.
+################################################################################
+
+Describe "BizTalk Build" {
+    BeforeAll {
+        $testSlnUri = "https://aka.ms/DevOpsAgentBuildSmokeTest.zip"
```

Comment:
Usually, we don't include such tests into the build process. Otherwise, the build time will be tragically increased.

During image generation, we run only very simple tests like: check that registry branch is created, folder exists and etc.

For this complex tests, we use https://github.com/microsoft/azure-pipelines-canary . Those tests are run during image deployment instead of image-generation. We can help to move this test there

Reply:
If it is not allowed, please help to move them to the other place, and I can create some other ones to verify the registry and folder existence instead.

Reply:
Add new commit to use registry/folder/files check instead of doing actually project build.